openlibrary/templates/home/stats.html

```diff
@@ -15,7 +15,7 @@ <h2 class="inline">$_('Around the Library')</h2>
         </div>
 
         <div class="statschart">
-          <div class="chartShow" onClick="window.location.href='/recentchanges';" title="$_('How many new Open Library members have we welcomed?')">
+          <div class="chartShow" onClick="window.location.href='//openlibrary.org/stats';" title="$_('How many new Open Library members have we welcomed?')">
             <div id="members-graph" style="width:150px;height:60px;"></div>
             <a href="/recentchanges" title="$_('How many new Open Library members have we welcomed?')"><span class="ticks">$:commify(stats["members"].get_summary(28))</span><span class="label">$_('New Members')</span></a>
```

Comment:
Ah; while testing, I found that it looks like it needs to be updated in 2 places (boy, does this code need a refactor!)

```suggestion
          <div class="chartShow" onClick="window.location.href='/stats';" title="$_('How many new Open Library members have we welcomed?')">
            <div id="members-graph" style="width:150px;height:60px;"></div>
            <a href="/stats" title="$_('How many new Open Library members have we welcomed?')"><span class="ticks">$:commify(stats["members"].get_summary(28))</span><span class="label">$_('New Members')</span></a>
```